project_report/tex/design.tex

```diff
@@ -0,0 +1,21 @@
+\section{Design}
+
+\subsection{Database Design}
+The project should have a database to store data. 
+The data that should be stored in the database is a batch. 
+You can see how the data should be stored in / 
+configured in the database in figure \ref{figure:eer_diagram_batch}.
+The data is useful for making a complete system for Refslevbæk Bryghus A/S, 
+then they can make sure to have data about the batch they made. 
+
+The Batch's database is used in the system to keep track of batch's.
+For example, if a person would like to know what happened then the beer got produced,
+like what temperature was it then the temperature was highest, Did that ruin it.
```

Comment:
"... like what temperature was it then the temperature was highest,"
change to:
"... like what the highest temperature was during the production ..."


"... , Did that ruin it."
Change to:
"... , did that have a negative effect on the beer."